plugins/modules/nmcli.py

```diff
@@ -2607,6 +2645,11 @@ def main():
             wireguard=dict(type='dict'),
             vpn=dict(type='dict'),
             transport_mode=dict(type='str', choices=['datagram', 'connected']),
+            #WireGuard peer attributes
+            wgpeer_public_key=dict(type='str', required=False),
+            wgpeer_allowed_ips=dict(type='str', required=False),
+            wgpeer_endpoint=dict(type='str', required=False),
+            wgpeer_persistent_keepalive=dict(type='int', required=False),
```

Comment:
This also does not match the documentation.

Reply:
Thanks for the update fixed these 
```
            wireguard=dict(type='dict', suboptions=dict(
                                wgpeer_public_key=dict(type='str', required=False),
                                wgpeer_allowed_ips=dict(type='str', required=False),
                                wgpeer_endpoint=dict(type='str', required=False),
                                wgpeer_persistent_keepalive=dict(type='int', required=False))),
```